ui/raidboss/data/05-shb/eureka/delubrum_reginae.ts

```diff
@@ -326,10 +326,6 @@ const triggerSet: TriggerSet<Data> = {
           console.error(`Iron Splitter: null data`);
           return;
         }
-        if (!seekerData.combatants) {
-          console.error(`Iron Splitter: null combatants`);
-          return;
-        }
```

Comment:
This check seems to exist randomly for only some of the getCombatants calls and is pointless, the actual handler in OverlayPlugin always returns an array here.

Reply:
Yeah, there's a lot of boiler plate here that probably shouldn't be copy pasta'd.  I sometimes wonder if we should have a get combatants promise helper.  Something that has the expected number of things that are returned, maybe even stores it in `data.combatantData` or something so all the logic moves out of the promise.  I don't know.  Just thinking out loud.